rviz_rendering/include/rviz_rendering/point_cloud.hpp

```diff
@@ -164,47 +172,47 @@ class PointCloud : public Ogre::MovableObject
-  void setCommonDirection( const Ogre::Vector3& vec );
+  void setCommonDirection(const Ogre::Vector3 & vec);
   /// See Ogre::BillboardSet::setCommonUpVector
-  void setCommonUpVector( const Ogre::Vector3& vec );
+  void setCommonUpVector(const Ogre::Vector3 & vec);
 
   /// set alpha blending
   /// @param alpha global alpha value
   /// @param per_point_alpha indicates that each point will have an individual alpha value.
   ///                        if true, enables alpha blending regardless of the global alpha.
-  void setAlpha( float alpha, bool per_point_alpha = false );
+  void setAlpha(float alpha, bool per_point_alpha = false);
 
-  void setPickColor(const Ogre::ColourValue& color);
+  void setPickColor(const Ogre::ColourValue & color);
   void setColorByIndex(bool set);
 
-  void setHighlightColor( float r, float g, float b );
+  void setHighlightColor(float r, float g, float b);
 
-  virtual const Ogre::String& getMovableType() const { return sm_Type; }
-  virtual const Ogre::AxisAlignedBox& getBoundingBox() const;
+  virtual const Ogre::String & getMovableType() const {return sm_Type;}
+  virtual const Ogre::AxisAlignedBox & getBoundingBox() const;
   virtual float getBoundingRadius() const;
-  virtual void getWorldTransforms( Ogre::Matrix4* xform ) const;
-  virtual unsigned short getNumWorldTransforms() const { return 1; }
-  virtual void _updateRenderQueue( Ogre::RenderQueue* queue );
-  virtual void _notifyCurrentCamera( Ogre::Camera* camera );
-  virtual void _notifyAttached(Ogre::Node *parent, bool isTagPoint=false);
+  virtual void getWorldTransforms(Ogre::Matrix4 * xform) const;
+  virtual uint16_t getNumWorldTransforms() const {return 1;}
+  virtual void _updateRenderQueue(Ogre::RenderQueue * queue);
+  virtual void _notifyCurrentCamera(Ogre::Camera * camera);
+  virtual void _notifyAttached(Ogre::Node * parent, bool isTagPoint = false);
 #if (OGRE_VERSION_MAJOR >= 1 && OGRE_VERSION_MINOR >= 6)
-  virtual void visitRenderables(Ogre::Renderable::Visitor* visitor, bool debugRenderables);
+  virtual void visitRenderables(Ogre::Renderable::Visitor * visitor, bool debugRenderables);
 #endif
 
-  virtual void setName ( const std::string& name ) { mName = name; }
+  virtual void setName(const std::string & name) {mName = name;}
 
 private:
-
   uint32_t getVerticesPerPoint();
-  PointCloudRenderablePtr createRenderable( int num_points );
+  PointCloudRenderablePtr createRenderable(int num_points);
   void regenerateAll();
   void shrinkRenderables();
 
   Ogre::AxisAlignedBox bounding_box_;       ///< The bounding box of this point cloud
   float bounding_radius_;                   ///< The bounding radius of this point cloud
 
   typedef std::vector<Point> V_Point;
-  V_Point points_;                          ///< The list of points we're displaying.  Allocates to a high-water-mark.
+  V_Point points_;                          ///< The list of points we're displaying.
+  ///< Allocates to a high-water-mark.
```

Comment:
I don't this comment should go after the variable it is describing, instead move the whole trailing comment to the line before the member variable.